### ISSUE TITLE
Inconsistent results, shows only 1-2 taskbar icons, and occasionally only spawns on one screen.

### ISSUE THREAD

**Issue by Lemagex** [ChrisAnd1998/TaskbarXI#193]
Inconsistent results, shows only 1-2 taskbar icons, and occasionally only spawns on one screen. Restarting multiple times will change the taskbar count, and will sometimes randomly change which screen it works on (or both). 

Windows 11 23H2 22631.2506
Home/Standard, x64
Intel CPU

**Comment by beuh0623** [comment]
Yeah, I got that too. Devs probably aren't doing anything to fix it. Just uninstall

**Comment by reta-yon** [comment]
yea same thing keeps happening to me now its getting worse, can't even see most of anything going on, it was showing 1 or 2 apps before now it just shows an empty space sometimes thick sometimes thin sometimes NONE 
![Screenshot 2023-11-17 054029](https://github.com/ChrisAnd1998/TaskbarXI/assets/147881692/7ec56c03-2772-49b7-ae09-ce8583f8fdc2)

**Comment by reta-yon** [comment]
i hope the devs look into this fr, here's my device info, ty
![Screenshot 2023-11-17 054806](https://github.com/ChrisAnd1998/TaskbarXI/assets/147881692/3c8ef231-221f-4487-b2c4-c7b950860fd2)

**Comment by Vyxeaa** [comment]
Same thing here, it's completely unusable.. Such a shame, because I really enjoyed this. I hope it gets fixed. :(

**Comment by william-joel** [comment]
it really sucks, i've got the same issues

**Comment by HarmlessPotato596** [comment]
hope this gets fixed

**Comment by LightCore00** [comment]
I think it's because of the operating system's features, I tried several similar programs, and they all have the same problems
